Sources/PackageDescription4/Target.swift

```diff
@@ -11,6 +11,12 @@
 /// The description for an individual target.
 public final class Target {
 
+    public enum TargetType: String {
```

Comment:
We don't want this to be public, do we?

Reply:
No option, need access in other targets :(

Reply:
Ah, because we deserialize into the same types... :/

Reply:
I mean we also need to access the types in other targets.